test/runtime/Policies.go

```diff
@@ -951,22 +952,40 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 	Context("TestsEgressToHost", func() {
 		hostDockerContainer := "hostDockerContainer"
 		hostIP := "10.0.2.15"
+		otherHostIP := ""
 
 		BeforeAll(func() {
 			By("Starting httpd server using host networking")
 			res := vm.ContainerCreate(hostDockerContainer, helpers.HttpdImage, helpers.HostDockerNetwork, "-l id.hostDockerContainer")
 			res.ExpectSuccess("unable to start Docker container with host networking")
+
+			By("Detecting host IP in world CIDR")
+			res = vm.Exec("ip route | grep '192.168' | head -n 1")
+			res.ExpectSuccess("unable to detect host IP for CIDR test")
+			routeParams := strings.Split(res.Output().String(), " ")
+			for _, param := range routeParams {
+				if net.ParseIP(param) != nil {
+					otherHostIP = param
+					break
+				}
+			}
+			Expect(otherHostIP).To(Not(Equal("")))
```

Comment:
`Expect(otherHostIP).NotTo(Equal(""))`

I prefer to use `NotTo` is a bit simpler IMHO.